capstone/src/main/java/com/google/univiz/CollegeData.java

```diff
@@ -0,0 +1,85 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/** CollegeData is a class representing a single university/college/institution. */
+@AutoValue
+abstract class CollegeData {
+
+  abstract CollegeId id();
+
+  abstract String name();
+
+  abstract String city();
+
+  /**
+   * Returns a boolean that tells whether the institution is the main campus or not, true or false.
+   */
+  abstract boolean isMainCampus();
+
+  abstract double latitude();
+
+  abstract double longitude();
+
+  /**
+   * Returns an enum CarnegieSizeDegree that represents the institution's carnegie size degree, or
+   * the highest degree certificate available at that institution, from not applicable to
+   * exclusively graduate/professional.
+   */
+  abstract CarnegieSizeDegree carnegieSizeDegree();
+
+  abstract double admissionRate();
+
+  abstract double avgSat();
+
+  abstract int numOfUndergrads();
+
+  abstract int avgCost();
+
+  abstract double ratioOfMen();
+
+  abstract double ratioOfWomen();
+
+  /**
+   * The CollegeData builder has some set default values. The avgSat and admissionRate values are
+   * estimated averages as of 2020.
+   */
+  static Builder builder() {
+    return new AutoValue_CollegeData.Builder()
+        .setCarnegieSizeDegree(CarnegieSizeDegree.NOT_AVAILABLE)
+        .setAvgSat(1059.0)
+        .setAdmissionRate(0.65);
```

Comment:
What are the meaning of these defaults? Can't we just omit them?

Reply:
these are averages i found. i used these because i didnt want it to say that the admission rate would be 0 or average sat would be 0 for a particular college.